utilities/bash/dot_profile

```diff
@@ -1613,7 +1613,7 @@ function sixdeskRenewKerberosToken(){
         local __lKlist=true
     fi
     sixdeskmess -1 " --> kinit;"
-    multipleTrials "kinit -R ; local __exit_status=\$?" "[ \$__exit_status -eq 0 ]"
+    multipleTrials "kinit -R ; local __exit_status=\$?; aklog -c cern.ch -k CERN.CH" "[ \$__exit_status -eq 0 ]"
```

Comment:
similarly to `control.sh`, I would keep `kinit` embedded in the `multipleTrials` algorithm, with the check of its success, and then have the `aklog` command, again as argument to the `multipleTrials` algorithm and a check afterwards